pkg/runtime/ecosystem/dotnet.go

```diff
@@ -0,0 +1,113 @@
+package ecosystem
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/unarchiver"
+
+	"github.com/ActiveState/cli/pkg/buildplan"
+)
+
+type DotNet struct {
+	runtimePath string
+	nupkgDir    string
+}
+
+func (e *DotNet) Init(runtimePath string, buildplan *buildplan.BuildPlan) error {
+	e.runtimePath = runtimePath
+	e.nupkgDir = filepath.Join("usr", "nupkg")
```

Comment:
The default .NET package registry lives in the user's _~/.nuget/packages/_ directory. We can override this with the `$NUGET_PACKAGES` environment variable, and we'll point it to this directory: _<runtime>/usr/nupkg/_.